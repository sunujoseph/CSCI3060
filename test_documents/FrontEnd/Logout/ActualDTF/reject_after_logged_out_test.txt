00 admin1          AA 150.780000
00 admin1          AA 150.780000
