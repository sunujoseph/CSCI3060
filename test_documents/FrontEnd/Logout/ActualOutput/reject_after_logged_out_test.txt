Enter Username: 
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Not Logged In
Error: Invalid Input
Error: Invalid Input
Enter Username: 
