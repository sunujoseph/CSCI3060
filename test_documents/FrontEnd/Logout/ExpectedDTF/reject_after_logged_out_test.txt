00 admin1          AA 000150.78
00 admin1          AA 000150.78
