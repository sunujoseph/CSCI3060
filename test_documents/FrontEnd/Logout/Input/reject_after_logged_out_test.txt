login
admin1
logout
logout
create
delete
advertise
bid
refund
addcredit
login
admin1
logout
