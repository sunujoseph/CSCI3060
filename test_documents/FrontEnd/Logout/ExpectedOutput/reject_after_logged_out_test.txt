Enter Username:
Logged Out
Error: Not logged in
Error: Not logged in
Error: Not logged in
Error: Not logged in
Error: Not logged in
Error: Not logged in
Error: Not logged in
Enter Username: 
Logged Out
